docs/private-app-what.md

```diff
@@ -0,0 +1,15 @@
+## What is a Private App
+
+Private apps allow you to use the Gateway’s APIs to access specific data from your Gateway Team. You can define what features/services each Private App is able to access from the management page. An access token is generated called the “ClientID” which must be sent with each request to the Gateway APIs. 
```

Comment:
Does the user need both the AppID and ClientID for each request to the Gateway APIs?

Reply:
Have rephrased. in next merge